docs/app/docs/ide_integration/eclipse.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Eclipse integration
+---
+
+
+## Java
+Eclipse IDE is mostly used for development of Java projects. Therefore, in this guide we cover setting up a Java environment that's created inside devbox shell to work with Eclipse IDE.
+
+### Setting up Devbox shell
```

Comment:
Out of curiosity, does this get automatically configured if you start Eclipse from within your Devbox Shell?

Reply:
Yes, I tested it and it does. I'll add that to the doc as well as an alternative.